module#2/src/constants.ts

```diff
@@ -17,5 +17,9 @@ export const dataBase = {
 };
 
 export const ERROR_MESSAGE = {
-  USER_ALREADY_EXIST: "A user with this login already exists."
+  USER_ALREADY_EXIST: "A user with this login already exists.",
+  USER_DOES_NOT_EXIST: "The requested user does not exist."
 };
+
+export const userDeletedMessage = (id: string) => `User with the id: ${id} has been deleted.`;
+export const userAlreadyDeletedMessage = (id: string) => `User with the id: ${id} already deleted.`;
```

Comment:
What is the difference between two? Wording? Past Perfect and some Perfect tenses? 😊

Reply:
I am using the first sentence after accomplished `delete` operation, the second if user is already deleted.  Yes, we don't delete user at all, just change flag `isDeleted`. I can add extra information might, but maybe it doesn't make sense. Maybe in  next task we will remove at all. I don't know.